app/src/main/java/com/kabouzeid/gramophone/ui/activities/AlbumDetailActivity.java

```diff
@@ -327,6 +388,7 @@ public void setStatusbarColor(int color) {
     private void setAlbum(Album album) {
         this.album = album;
         loadAlbumCover();
+        loadWiki();
```

Comment:
I just saw that currently the artist bio is always downloaded automatically. This shouldn't be the case. In this case we can fix this all together.